recipes/pymeshlab/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set name = "pymeshlab" %}
+{% set version = "2023.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/cnr-isti-vclab/PyMeshLab/archive/v{{ version }}.tar.gz
+    sha256: 0ef40886952d7521bd76b50d6d57de994cf8693a910456fa31dcf4b72e843c41
+  - url: https://github.com/cnr-isti-vclab/meshlab/archive/2dbd2f4b12df3b47d8777b2b4a43cabd9e425735.tar.gz
+    sha256: 9236ad0d27e6712be247148789737598058c8f332e0333a36f48c54d0a475061
+    folder: src/meshlab
+  - url: https://github.com/cnr-isti-vclab/vcglib/archive/6ac9e0c647a63e0e037813a1e92bd050d13efc85.tar.gz
+    sha256: aeb76137d29f1c7f19b1d82b57e4f347139d3d345533c35578f944af83eea29d
+    folder: src/meshlab/src/vcglib
+  - url: https://github.com/pybind/pybind11/archive/refs/tags/v2.11.2.tar.gz
+    sha256: c38f52f75a4e2af9a5bcff2368713101585acc06963891af5f03eaa7f4c04242
+    folder: src/pymeshlab/pybind11
+
+build:
+  number: 0
+  skip: true  # [py>310]
+  skip: true  # [osx]
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - {{ stdlib("c") }}
+    - {{ cdt('mesa-libgl-devel') }}          # [linux]
+    - {{ cdt('mesa-dri-drivers') }}          # [linux]
+    - {{ cdt('mesa-libegl-devel') }}         # [linux]
+    - {{ cdt('libselinux') }}                # [linux]
+    - {{ cdt('libxdamage') }}                # [linux]
+    - {{ cdt('libxfixes') }}                 # [linux]
+    - {{ cdt('libxxf86vm') }}                # [linux]
+    - {{ cdt('libxcb') }}                    # [linux]
+    - {{ cdt('libxext') }}                   # [linux]
```

Comment:
If all these X library are needed, you can just add them as conda host requirements, as they should all be available as proper conda packages.